core/src/Focusable.ts

```diff
@@ -1273,20 +1284,16 @@ export class FocusableAPI implements Types.FocusableAPI {
         ignoreGroupper?: boolean
     ): number {
         const ctx = RootAPI.getTabsterContext(this._tabster, element);
-        const currentModalizerId = ctx && ctx.root.getCurrentModalizerId();
+        if (this._isHidden(element)) {
```

Comment:
Should this be `isAccessible` too ? 
```suggestion
        if (this.isAccessible(element)) {
```